### PR TITLE
SC-16756 add release notes for sematext-agent 3.0.0

### PR DESCRIPTION
**Pull request by boratanrikulu**

### CODE REVIEW

**Comment by boratanrikulu** [comment]
It's still draft but mostly done, please check language and missing any changes.

File: docs/agents/sematext-agent/releasenotes.md [hunk 1]
```diff
@@ -35,6 +35,51 @@ NULL
 
 -->
 
+# Version 3.0.0
+
+Date: July 25, 2023
+
+## New Features
+
+- SC-5833 [**Kubernetes Components**](https://kubernetes.io/docs/concepts/overview/components/) monitoring integration: We are excited to announce that Sematext Agent now supports monitoring of critical Kubernetes components, including **API Server**, **CoreDNS**, **kube-proxy**, **Scheduler**, and **Etcd**. Gain deeper insights into your Kubernetes cluster's performance with this powerful integration.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:44]: @boratanrikulu let's add kubelet reports as well, you'll be notified by @yagogou as soon as we've update them.

**Review by yagogou — APPROVED** [review]
LGTM

File: docs/agents/sematext-agent/kubernetes/hostnetwork.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ Sematext Agent requires `hostNetwork: true` setting to monitor the control plane
 
 If you want to turn off `hostNetwork` access, it's important to note that you may lose the ability to monitor some control plane components. However, if you still want to proceed, here are the instructions based on the installation method you are using:
 
-### Helm
+### Helm Installation Method
```
Comment (otisg) [inline docs/agents/sematext-agent/kubernetes/hostnetwork.md:16]: Just XXXX Installation without "Method" is probably a bit better.

File: docs/agents/sematext-agent/releasenotes.md [hunk 3]
```diff
@@ -35,17 +35,72 @@ NULL
 
 -->
 
+# Version 3.0.0
+
+Date: July 25, 2023
+
+## New Features
+
+- [**Kubernetes Components**](https://kubernetes.io/docs/concepts/overview/components/) monitoring integration: We are excited to announce that Sematext Agent now supports monitoring of critical Kubernetes components, including **API Server**, **CoreDNS**, **kube-proxy**, **kubelet**, **Scheduler**, and **Etcd**. Gain deeper insights into your Kubernetes cluster's performance with this powerful integration. Check out [Kubernetes Monitoring Integration](https://sematext.com/docs/integration/kubernetes/) for the details.
```
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:44]: for the details ==> for details

File: docs/agents/sematext-agent/releasenotes.md [hunk 4]
```diff
@@ -35,17 +35,72 @@ NULL
...
+- Varnish Cache logs parsing support: With this update, Sematext Agent can now parse fields from your Varnish Cache log messages. Stay tuned for the upcoming release of Varnish Cache logs integration in Sematext Cloud, enhancing your logging experience further.
+
+## Improvements
+
+- Improved discovery for OpenSearch integration: We have enhanced the OpenSearch integration by preventing the discovery of `.json` files, which are not supported. Now, Sematext Agent will only discover and support `.log` files for OpenSearch integration, ensuring a smoother experience.
+
+## Bug Fixes
+
+- SC-16613 Fixed incorrect `Not Configured` status for the log shipper: We have resolved an issue that was causing the log shipper to display an incorrect status message. With this fix, you will now receive accurate log shipper status updates.
```
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:53]: an issue ==> the issue
With this fix, you will now receive accurate log shipper status updates. ==> not necessary

File: docs/agents/sematext-agent/releasenotes.md [hunk 5]
```diff
@@ -35,17 +35,72 @@ NULL
...
+> helm search repo sematext
+NAME                   	CHART VERSION	APP VERSION	DESCRIPTION                                       
+sematext/sematext-agent	1.2.1        	1.0        	Helm chart for deploying Sematext Agent and Log...
+```
+
+### `hostNetwork` Setting Usage
+
+In Kubernetes, the `hostNetwork` is a configuration option that allows a container to share the network namespace of its host node. Sematext Agent requires `hostNetwork: true` setting to monitor the control plane components. By default, the `hostNetwork` setting is set to `true` starting from version 3.0.0 of Sematext Agent.
+
+If you want to turn off `hostNetwork` access, check [our page about hostnetwork](https://sematext.com/docs/agents/sematext-agent/kubernetes/hostnetwork/).
```
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:87]: our page ==> our docs

File: docs/agents/sematext-agent/releasenotes.md [hunk 6]
```diff
@@ -35,17 +35,72 @@ NULL
...
+- MySQL logs parsing support: We have enhanced the log parsing capabilities of Sematext Agent with this update. Your log messages can now be parsed more effectively, leading to improved visibility and analysis.
 
 ## Improvements
 
-- SC-15612 Improved container fields: Sematext Agent now provides more comprehensive fields for containers in Kubernetes. We have introduced the missing `container.type` field, allowing for better categorization of containers. Additionally, we have added the new `container.image.tag` field, providing valuable information about container images for improved management and identification.
+- Improved container fields: Sematext Agent now provides more comprehensive fields for containers in Kubernetes. We have introduced the missing `container.type` field, allowing for better categorization of containers. Additionally, we have added the new `container.image.tag` field, providing valuable information about container images for improved management and identification.
```
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:103]: Might be handy to link to container tags section in the docs?

File: docs/agents/sematext-agent/releasenotes.md (hunk 4 above)
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:53]: No need for SC-16613.  We stopped including those.

Comment (boratanrikulu) [reply docs/agents/sematext-agent/releasenotes.md:53]: Do you mean ticket number or whole note for `Not Configured`?

Comment (otisg) [reply docs/agents/sematext-agent/releasenotes.md:53]: Just the ticket number

File: docs/agents/sematext-agent/releasenotes.md [hunk 7]
```diff
@@ -35,67 +35,126 @@ NULL
...
+```
+
+Ensure that you have at least **1.2.1** version for the `sematext-agent` Helm package.
+```
+> helm search repo sematext
+NAME                   	CHART VERSION	APP VERSION	DESCRIPTION                                       
+sematext/sematext-agent	1.2.1        	1.0        	Helm chart for deploying Sematext Agent and Log...
+```
+
+After updating the Helm chart, follow the agent upgradation steps from Sematext Cloud.
```
Comment (otisg) [inline docs/agents/sematext-agent/releasenotes.md:85]: upgradation ==> upgrade

**Review by otisg — APPROVED** [review]
Did only a very quick scan.  One small comment.  LGTM.

**Pull request merged**